modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/util/DDMStructureUtil.java

```diff
@@ -42,13 +43,8 @@ public static List<String> getAvailableLanguageIds(
 		Set<Locale> locales = LanguageUtil.getAvailableLocales(
 			themeDisplay.getSiteGroupId());
 
-		Stream<Locale> stream = locales.stream();
-
-		return stream.map(
-			LanguageUtil::getLanguageId
-		).collect(
-			Collectors.toList()
-		);
+		return new ArrayList<>(
```

Comment:
@slnn This `new ArrayList<>()` should be redundant.

Reply:
Oh,  got it, transform(Collection<T>, UnsafeFunction<T, R, E>) always return a List instead of a Collection..

Reply:
Resend in https://github.com/liferay-core-infra/liferay-portal/pull/3850